src/test/java/io/tiledb/java/api/GroupTest.java

```diff
@@ -131,7 +131,7 @@ public void membersTest() throws Exception {
 
     // remove a member
     group.reopen(ctx, TILEDB_WRITE);
-    group.removeMember("array2Name");
+    group.removeMember("array2");
```

Comment:
Shouldn't this test have failed before?

Reply:
I believe it should work both ways right? The method from core has changed to allow for either the uri or the group name.

Reply:
I added both use cases to avoid confusion. See my latest commit.